extension/src/popup/helpers/__tests__/decodeHash.test.js

```diff
@@ -4,34 +4,38 @@ import { decodeMemo } from "../parseTransaction";
 describe("decodeMemo", () => {
   it("should return a memo id", () => {
     const MEMO = {
```

Comment:
Should we use the actual Memo class here instead of stubbing it? I think typically when we see the properties that are prefixed with an underscore, like `_type` or `_value`, the convention is to use the class getter to access it as they intend the field to be private to the class.
https://github.com/stellar/js-stellar-base/blob/cf370f4e627e2b0e296ceceaebd0164dede396bd/src/memo.js#L64